### PR TITLE
[DOCU-1650][DOCU-1653] Konnect deploy Aug31

### PR DESCRIPTION
**Pull request by lena-larionova**
### Summary
* Entry in changelog to introduce support for 2.5
* Update version in sample codeblocks for runtime setup
* Remove `lua_ssl_trusted_certificate` from the doc due to changes made in https://github.com/kong/kong/pull/7044.
 
### Reason
Konnect CP has been updated to run on Gateway 2.5.0.1.
https://konghq.atlassian.net/browse/DOCU-1650
https://konghq.atlassian.net/browse/DOCU-1653

### Testing
Changelog entry: 
https://deploy-preview-3163--kongdocs.netlify.app/konnect/updates

Versions and `lua_ssl_trusted_certificate` updated in:
https://deploy-preview-3163--kongdocs.netlify.app/konnect/runtime-manager/gateway-runtime-docker/
https://deploy-preview-3163--kongdocs.netlify.app/konnect/runtime-manager/gateway-runtime-conf/
https://deploy-preview-3163--kongdocs.netlify.app/konnect/runtime-manager/gateway-runtime-kubernetes/

### CODE REVIEW

**Review by HCloward — APPROVED** [review]
Everything looks good to me. A couple of things I noticed that shouldn't hold up the merge, but worth a mention, the 2nd end note for some reason appears to the left of the first one and not underneath. Perhaps needs an html </br> tag? Also, in the code samples we recently agreed that instead of using <> to hold placeholder text we use {} but I understand that might be outside the scope for this PR.

**Comment by lena-larionova** [comment]
>  The 2nd end note for some reason appears to the left of the first one and not underneath. Perhaps needs an html
> tag?

Nice catch, I'll fix that before merge.

> Also, in the code samples we recently agreed that instead of using <> to hold placeholder text we use {} but I understand that might be outside the scope for this PR.

Yeah, out of scope for this PR, but will revisit.

**Pull request merged**
